action.yml

```diff
@@ -12,3 +29,4 @@ runs:
     - secret
     - scan
     - ci
+    - ${{ inputs.args }}
```

Comment:
Because of the way ggshield argument parser works, arguments like `--json` or `--banlist-detector` must be placed between `scan` and `ci`, so I think `args` must be changed to something like this:

```yaml
args:
  - ggshield
  - -v
  - secret
  - scan
  - ${{ inputs.args }}
  - ci
```

But I haven't verified if it works (and I am actually not sure how to try it)

Reply:
Alright. Is it a good idea if I create the same PR on this (soon to be deprecated) [repository](https://github.com/GitGuardian/ggshield-action) to try your suggested change?

Reply:
Since we want to deprecate it, I think we should leave it as is for now. When we are confident the action in this repository works as expected we can add a deprecation message to the ggshield-action repository (both in the README and in the action code itself).

Regarding testing the changes: if I am not mistaken you should be able to test your branch using `uses: GitGuardian/ggshield@list-gh-action-mktplace` (but maybe you already knew that :) )